src/ExcelReader.cs

```diff
@@ -46,84 +21,7 @@ public ExcelReader(String filename)
                 {
                     if (File.Exists(filename))
                     {
-                        string strConn = ExcelProvider.GetOLEDB12ConnectionString(Filename);
-
-                        if (filename.EndsWith(".xls", StringComparison.OrdinalIgnoreCase))
-                        {
-                            strConn = ExcelProvider.GetOLEDB4ConnectionString(Filename);
-                        }
-
-                        DataSet ds = new DataSet();
-
-                        using (OleDbConnection conn = new OleDbConnection(strConn))
-                        {
-                            try
-                            {
-                                conn.Open();
-                            }
-                            catch (Exception)
-                            {
-                                throw;
-                            }
-
-                            DataTable schemaTable = conn.GetOleDbSchemaTable(OleDbSchemaGuid.Tables, new object[] { null, null, null, "TABLE" });
-
-                            foreach (DataRow schemaRow in schemaTable.Rows)
-                            {
-                                string sheet = schemaRow["TABLE_NAME"].ToString();
-
-                                if (!sheet.EndsWith("_") && sheet.Contains("$"))
-                                {
-                                    try
-                                    {
-                                        OleDbCommand cmd = new OleDbCommand("SELECT * FROM [" + sheet + "]", conn);
-                                        cmd.CommandType = CommandType.Text;
-
-                                        DataTable outputTable = new DataTable(sheet);
-                                        ds.Tables.Add(outputTable);
-                                        new OleDbDataAdapter(cmd).Fill(outputTable);
-                                        outputTable.Dispose();
-                                    }
-                                    catch (Exception ex)
-                                    {
-                                        throw new Exception(ex.Message + string.Format("Sheet: {0}.File.F{1}", sheet, Filename), ex);
-                                    }
-                                }
-                            }
-
-                            conn.Close();
-                            conn.Dispose();
-                        }
-
-                        ExcelSet = ds;
-
-                        foreach (DataTable dt in ExcelSet.Tables)
-                        {
-                            List<DataRow> deleteRows = new List<DataRow>();
-                            dt.TableName = dt.TableName.Replace("$", String.Empty);
```

Comment:
Looks like this is missing in a new code, so maybe we should keep it to not break smth?